cmake/LLVM.cmake

```diff
@@ -215,11 +215,10 @@ set(POCL_LLVM_LIBS ${LLVM_LIBFILES})
 
 # this needs to be done with LLVM_LIB_MODE because it affects the output
 run_llvm_config(LLVM_SYSLIBS --system-libs ${LLVM_LIB_MODE})
-string(STRIP "${LLVM_SYSLIBS}" LLVM_SYSLIBS)
 # TODO this hack is required for MinGW. Without it,
 # Clang link test reports missing symbols like GetFileVersionInfoSizeW
 if(MINGW)
-  set(LLVM_SYSLIBS "${LLVM_SYSLIBS} -lversion" CACHE STRING "llvm's syslibs" FORCE)
+  list(APPEND LLVM_SYSLIBS "-lversion")
```

Comment:
Might need `string(REPLACE " " ";" LLVM_SYSLIBS "${LLVM_SYSLIBS}")` before this line. I think  the LLVM_SYSLIBS is populated with whitespace separated items so the appending here probably results in two element list instead of N + 1 elements.

Reply:
If we need string operations anyway, let's just keep it as a string and retain the original `string(strip)` (which was there before my commit already, just in the wrong place)?

Reply:
OK, fair enough.